lib/internal/bootstrap_node.js

```diff
@@ -304,6 +306,40 @@
     };
   }
 
+  function setupProcessICUVersions() {
+    const icu = process.binding('icu');
+    if (icu) {
+      const object = process.versions;
+      // if ICU is present, fetch additional versions
+      // TODO(srl295): should this be a comma separated list or something else?
+      const versionTypes = icu.getVersion().split(',');
+      versionTypes.forEach((name) => {
+        // Copied from module.js:addBuiltinLibsToObject
+
+        Object.defineProperty(object, name, {
+          get: () => {
+            const realValue = icu.getVersion(name);
+
+            // Disable the current getter/setter and set up a new
+            // property.
+            delete object[name];
+            Object.defineProperty(object, name, {
+              value: realValue,
+              writable: false,
+              // configurable: false,
+              enumerable: true
+            });
+
+            return realValue;
+          },
+          // set: setReal,
```

Comment:
@srl295 what part did you remove? What I meant was the `// set: setReal,` comment, because that’s a leftover from copying, and it seems a bit confusing given that there is no `setReal` defined here